lib/index.js

```diff
@@ -172,6 +172,7 @@ Segment.prototype.normalize = function(msg) {
   msg.userId = msg.userId || user.id();
   msg.anonymousId = user.anonymousId();
   msg.sentAt = new Date();
+  extend(msg, this._extras);
```

Comment:
We'll want to attach this data under the `__metadata__` key; otherwise this will pollute the top-level object with stray keys (and potentially overwrite data)